src/main/java/org/opensearch/performanceanalyzer/rca/framework/metrics/WriterMetrics.java

```diff
@@ -145,7 +145,18 @@
                     Statistics.SUM)),
 
     STALE_METRICS("StaleMetrics", "count", Arrays.asList(Statistics.COUNT)),
-    ;
+
+    // This metrics indicates that the write file is not created.
+    // When there is no metrics to be written, the creation of writer file is skipped.
```

Comment:
nit: Can we remove this line ? The metrics here might be confusing to person reading as here we imply to the event log metric but the line above refers to the service log metric.

Reply:
Right removed the second line.